From 5f71060f11e463c2446241a60fdd8ad297edcaff Mon Sep 17 00:00:00 2001
From: Patrick Mooney <pmooney@pfmooney.com>
Date: Fri, 8 Sep 2017 23:59:09 +0000
Subject: [PATCH] OS-6339 lint error due to xsaveopt merge Reviewed by: Jerry
 Jelinek <jerry.jelinek@joyent.com> Approved by: Jerry Jelinek
 <jerry.jelinek@joyent.com>

---
 usr/src/uts/intel/ia32/ml/float.s | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/usr/src/uts/intel/ia32/ml/float.s b/usr/src/uts/intel/ia32/ml/float.s
index 0d33365764..f154a96851 100644
--- a/usr/src/uts/intel/ia32/ml/float.s
+++ b/usr/src/uts/intel/ia32/ml/float.s
@@ -277,10 +277,6 @@ void
 xsave_ctxt(void *arg)
 {}
 
-void
-xsaveopt_ctxt(void *arg)
-{}
-
 /*ARGSUSED*/
 void
 xsaveopt_ctxt(void *arg)
-- 
2.21.0

